1906-10-29.xml

```diff
@@ -0,0 +1,1384 @@
+                        </row>
+                        <row>
+                            <cell><name>Pr. Reg. Luitpold</name></cell>
+                            <cell><measure unit="ton" quantity="8965">6283</measure> Tons</cell>
+                            <cell>about <date when="1905-07-10">29 October</date></cell>
+                        </row>
+                        <row>
+                            <cell><name>Prinz E. Friedrich</name></cell>
+                            <cell><measure unit="ton" quantity="8965">8965</measure> Tons</cell>
+                            <cell>about <date when="1905-07-10">12 November</date></cell>
+                        </row>
+                        <row>
+                            <cell cols="3">For AUSTRALIA via SUEZ, ADEN, COLOMBO.</cell>
+                        </row>
+                        <row>
+                            <cell><name>Barbarossa</name></cell>
+                            <cell><measure unit="ton" quantity="10915">10915</measure> Tons</cell>
+                            <cell>about <date when="1905-09-24">18 November</date>.</cell>
+                        </row>
+                        <row>
+                            <cell><name>Scharnhorst</name></cell>
+                            <cell><measure unit="ton" quantity="10915">8131</measure> Tons</cell>
+                            <cell>about <date when="1905-09-24">16 December</date>.</cell>
+                        </row>
+                    </table>
+                    <p>FOR FURTHER PARTICULARS APPLY TO THE AGENTS OF THE</p>
+                    <p>NORDDEUTSCHER LLOYD at Cairo, Alexandria, Port-Said and Suez.</p>
+                    <p>OTTO STERZING, Agent In Cairo, Opera Square.</p>
+                    <p>C. H. SCHOELLER, Agent In Alexandria, Cleopatra Lane.</p>
+                    <p>Messrs. THOS. COOK &amp; SON (Egypt) LTD., and CARL STANGENS REISEBUREAN are
+                        anthorised to sell tickets in CAIRO and ALEXANDRIA, <measure type="indexNo">31-8-905</measure></p>
+                </div><div type="advert" xml:id="deg-ad-als02">
+                    <head>Austrian Lloyd’s Steam Navigation</head>
+                    <p>Alexandria-Brindisi-Trieste (Venice).</p>
+                    <p>Weekly Express Mail Service. Steamers leave Alexandria every Saturday at 4
+                        p.m. arrive at Brindisi, Tuesday 5 a.m. in time for express to Milan,
+                        Lucerne, Paris, Vienna, Berlin, and London, leaving Brindisi 7 a.m. Arrival
+                        Trieste Wednesday about 11 a.m. in connection with Train de Luxe
+                        Trieste-Vienna-Ostende. Passengers reach London every Friday 4.50 p.m.</p>
+                    <table rows="3" cols="8">
+                        <row>
+                            <cell><date when="1906-10-06">Oct. 6</date></cell>
+                            <cell>4 p.m.</cell>
+                  
```

Comment:
Move this line and the next two lines, which start the advertising section, just after the masthead.